libs/s25main/ingameWindows/iwTransport.h

```diff
@@ -28,9 +29,15 @@ class GameCommandFactory;
 class iwTransport : public IngameWindow
 {
 private:
+    struct ButtonData
+    {
+        ITexture* sprite;
+        std::string tooltip;
+    };
     const GameWorldViewer& gwv;
     GameCommandFactory& gcFactory;
-    ITexture* TRANSPORT_SPRITES[14]; /// Einzelne Bilder für die Waren.
+
+    std::array<ButtonData, 14> buttonData;
```

Comment:
can we have a constant for the 14?

Reply:
(numButtons?)

Reply:
I did it the other way round: Using `buttonData.size` as `numButtons`, which IMO makes more sense: https://github.com/Return-To-The-Roots/s25client/pull/1301/files#diff-b140bedff094aa8fd2769a0841d341a4R66

But actually we should use `TransportOrders.size`... Adding a commit